### ISSUE TITLE
Mockyeah server filepaths not found

### ISSUE THREAD

**Issue by sibbl** [mockyeah/mockyeah#640]
I've just set up a plain simple npm project and ran `npm install @mockyeah/server`, which installed version 1.0.3.

Next to the `package.json`, I've created the following files:

index.js:
```js
const mockyeah = require("@mockyeah/server");
mockyeah.get("/helloworld", { filePath: "helloworld.json" });
```

helloworld.json
```json
{ "hello": "world" }
```

When starting and GETting `/helloworld`,  the following exception occurs:

```
Error: ENOENT: no such file or directory, open '...myprojectfolder\node_modules\helloworld.json'
```

What also doesn't work, is `{ filePath: __dirname + "/helloworld.json" }`.
The same issue happens when creating a fixtures directory and putting the JSON file in there.

I could fix it by prepending the following to my index.js:
```js
global.MOCKYEAH_ROOT = __dirname;
```

As far as I can see, the following file might be the issue. It goes 3 folders up, but from `node_modules/@mockyeah/server/lib` it should be 4 times `..`. https://github.com/mockyeah/mockyeah/blob/6113b9dfbd82858011aa101db8380d955b478122/packages/mockyeah/lib/relativeRoot.js#L5

**Comment by AndersDJohnson** [comment]
@sibbl Good catch, sorry about that, thanks for reporting! Are you interested in submitting a pull request with that fix?

I will let you know this project is not being actively maintained right now - you might look into something like [MSW](https://mswjs.io/) as a replacement.
